app/src/main/java/org/eyeseetea/malariacare/presenter/ReviewPresenter.java

```diff
@@ -0,0 +1,55 @@
+package org.eyeseetea.malariacare.presenter;
+
+import org.eyeseetea.malariacare.data.database.datasources.ValueLocalDataSource;
+import org.eyeseetea.malariacare.data.database.utils.Session;
+import org.eyeseetea.malariacare.domain.boundary.executors.IAsyncExecutor;
+import org.eyeseetea.malariacare.domain.boundary.executors.IMainExecutor;
+import org.eyeseetea.malariacare.domain.boundary.repositories.IValueRepository;
+import org.eyeseetea.malariacare.domain.entity.Value;
+import org.eyeseetea.malariacare.domain.usecase.AGetReviewValuesBySurveyIdUseCaseStrategy;
+import org.eyeseetea.malariacare.domain.usecase.strategies.GetReviewValuesBySurveyIdUseCaseStrategy;
+import org.eyeseetea.malariacare.layout.adapters.dashboard.ReviewScreenAdapter;
+import org.eyeseetea.malariacare.presentation.executors.AsyncExecutor;
+import org.eyeseetea.malariacare.presentation.executors.UIThreadExecutor;
+
+import java.util.List;
+
+public class ReviewPresenter implements ReviewScreenAdapter.onClickListener {
+
+    public interface ReviewView {
+        void showValues(List<Value> values);
+
+        void initListView();
+
+        void navigateToQuestion(String uId);
+    }
+
+    ReviewView view;
+
+    public ReviewPresenter() {
+    }
+
+    public void attachView(ReviewView reviewView) {
+        this.view = reviewView;
+        IMainExecutor mainExecutor = new UIThreadExecutor();
```

Comment:
The dependencies should not be created from the presenter, but from the composition root, as we do not have Dagger is still better to be created and assigned from the view (activity or fragment)